lib/pa_ess/utilities.ex

```diff
@@ -710,6 +741,7 @@ defmodule PaEss.Utilities do
   def audio_take(:upcoming_departures), do: "548"
   def audio_take(:upcoming_arrivals), do: "550"
   def audio_take(:is_now_arriving), do: "24055"
+  def audio_take(:is_passing_through), do: "929"
```

Comment:
Can we name this `:does_not_take_customers` so it matches the audio?

Reply:
Will change. I wasn't sure how important it was that these exactly match the words in the audio (since of course it's possible we could upload new audio for the same take in the future if we change the wording), and "passing through" matches how we refer to this concept elsewhere in the code.